clang/lib/DPCT/RulesLang/RulesLang.cpp

```diff
@@ -8511,6 +8521,22 @@ void DriverDeviceAPIRule::runRule(
             false);
       return;
     }
+  } else if (APIName == "cuDeviceGetP2PAttribute") {
+    if (!DpctGlobalInfo::usePeerAccess()) {
+      report(CE->getBeginLoc(), Diagnostics::API_NOT_MIGRATED, false, APIName);
+      return;
+    }
+    auto SecArg = CE->getArg(1);
+    if (auto DRE = dyn_cast<DeclRefExpr>(SecArg)) {
+      auto AttributeName = DRE->getNameInfo().getAsString();
```

Comment:
If a variable is used here, is the result as expected?

Reply:
Yes.
Also, modified the logic to account for the initialized value for a variable to emit a warning message for unsupported enum value if performance rank.